packages/vats/src/ibc.js

```diff
@@ -25,11 +25,6 @@ const DEFAULT_ACKNOWLEDGEMENT = '\x00';
  * @typedef {import('./bridge').BridgeHandler} BridgeHandler
  */
 
-/**
- * @template U
- * @typedef {import('@agoric/promise-kit').PromiseRecord<U>} PromiseRecord
```

Comment:
I suspect you were too aggressive in removing the definitions of several of these `PromiseRecord<T>` types.  Please search the file for occurrences of `PromiseRecord` (or `ERef`), because if you delete the explicit import, then these types default to `any`, which is a fate worse than death.

Okay, maybe not **that** bad, but still bad. :)

Reply:
Done!

I synced my vscode to my workspace typescript version as you suggest at https://github.com/Agoric/agoric-sdk/issues/3840#issuecomment-922374493

I then hovered over at least one `PromiseRecord` in every *.js file containing this identifier, and they all looked good. I also hovered over a random sampling of the occurrences of `ERef`. Both look exactly right.